browser.js

```diff
@@ -17,7 +18,17 @@ module.exports = function(blob, cb){
           if (err) return cb(err);
 
           var buf = typedToBuffer(typed);
-          cb(null, parse(buf), buf);
+          var obj;
+
+          if (buf[0] === 60) {
+            obj = plistParse(buf.toString());
```

Comment:
Yeah, something like `var openBrace = 60` or whatever is more appropriate, so it's easy to read but we don't need comments